ott/core/bar_problems.py

```diff
@@ -116,30 +121,37 @@ def add_slice_for_debiased(
 
   @property
   def flattened_y(self) -> Optional[jnp.ndarray]:
+    """Array of shape ``[num_measures * N, D]``."""
```

Comment:
technically this is only valid if all measures have the same size. Maybe *(N_1 + N_2 + ... + N_num_measures) ?